pkg/project/project.go

```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA             = "n/a"
 	name        string = "azure-operator"
 	source      string = "https://github.com/giantswarm/azure-operator"
-	version            = "4.1.0-dev"
+	version            = "4.1.0-refactoring-masters-instance"
```

Comment:
you probably don't want to merge this to master

Reply:
Yup, I'll remove it before merging :)